### PR TITLE
Standardize CI Workflow Using Kleros Scripts

### LINKED ISSUE #94: Set up continuous integration with Travis
- [x] Tests
- [ ] Linting
    - [x] Setup Solium
    - [ ] Fix Solium issues
    - [ ] Activate Solium
- [ ] Test coverage

### PR DESCRIPTION
**Pull request by ferittuncer**
Fixes #94 

The only problem is: 

Currently `security/no-inline-assembly` rule creates an error and `ExperimentalProxy.sol` uses inline assembly. Builds won't pass unless we remove inline assembly or alter the rules of Solium.

### CODE REVIEW

**Comment by epiqueras** [comment]
// solium-disable-line security/no-inline-assembly
// solium-disable-next-line security/no-inline-assembly

**Comment by ferittuncer** [comment]
Fixed and amended the commit.

File: .travis.yml [hunk 1]
```diff
@@ -1,27 +1,16 @@
 sudo: required
```
Comment (epiqueras) [inline .travis.yml]: Why do we need sudo?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,23 @@
...
 
+<p align="center">
+  <b style="font-size: 32px;">Kleros</b>
+</p>
+
+<p align="center">
+  <a href="https://standardjs.com"><img src="https://img.shields.io/badge/code_style-standard-brightgreen.svg" alt="JavaScript Style Guide"></a>
+  <a href="https://travis-ci.org/kleros/kleros"><img src="https://travis-ci.org/kleros/kleros.svg?branch=master" alt="Build Status"></a>
+  <a href="https://david-dm.org/kleros/kleros"><img src="https://david-dm.org/kleros/kleros.svg" alt="Dependencies"></a>
+  <a href="https://david-dm.org/kleros/kleros?type=dev"><img src="https://david-dm.org/kleros/kleros/dev-status.svg" alt="Dev Dependencies"></a>
```
Comment (epiqueras) [inline README.md]: These are the badges for `kleros/kleros`.

File: contracts/standard/arbitration/MultipleArbitrableTransaction.sol [hunk 3]
```diff
@@ -340,8 +340,13 @@ contract MultipleArbitrableTransaction {
         uint[] memory transactionIDsBigArr = new uint[](transactions.length);
         uint count = 0;
         for (uint i = 0; i < transactions.length; i++)
+        {
```
Comment (epiqueras) [inline contracts/standard/arbitration/MultipleArbitrableTransaction.sol]: Why did you add this?

File: contracts/standard/proxy/ExperimentalProxy.sol [hunk 4]
```diff
@@ -1,3 +1,5 @@
+// solium-disable security/no-inline-assembly
```
Comment (epiqueras) [inline contracts/standard/proxy/ExperimentalProxy.sol]: Best practice is to disable only in the line it is required.

File: README.md (hunk 2 above)
Comment (ferittuncer) [reply README.md]: Ooops, yeah.

File: .travis.yml (hunk 1 above)
Comment (ferittuncer) [reply .travis.yml]: We don't need actually. I needed it in my previous configuration and forgot to delete.

File: contracts/standard/arbitration/MultipleArbitrableTransaction.sol (hunk 3 above)
Comment (ferittuncer) [reply contracts/standard/arbitration/MultipleArbitrableTransaction.sol]: It was like this:
```
1        for (uint i = 0; i < transactions.length; i++)
2           if (transactions[i].seller == _address || transactions[i].buyer == _address)
3                transactionIDsBigArr[count++] = i;
```
Prettier gives an error:  
> 3:0      error      Only use indent of 8 spaces.  

Reducing indentation to 8 from 12 like this fixes the error:
```
1        for (uint i = 0; i < transactions.length; i++)
2           if (transactions[i].seller == _address || transactions[i].buyer == _address)
3           transactionIDsBigArr[count++] = i;
```

But this is a misleading indentation as line 3 should be one level more intended than line 2.

To fix it I added omitted braces.

Comment (epiqueras) [reply contracts/standard/arbitration/MultipleArbitrableTransaction.sol]: @ferittuncer 
The indentation was wrong.
You had 8, 11, and 16.
It should be 8, 12, and 16.

Comment (ferittuncer) [reply contracts/standard/arbitration/MultipleArbitrableTransaction.sol]: In the comment? Yes. 

But I still can't pass Prettier using 8 12 16.

Comment (ferittuncer) [reply contracts/standard/arbitration/MultipleArbitrableTransaction.sol]: I sent a commit to demonstrate.

Comment (ferittuncer) [reply contracts/standard/arbitration/MultipleArbitrableTransaction.sol]: Yeah, this one works. So it seems the bug happens when braces omitted multiple levels.

I'll fix this and squash all the commits. @epiqueras

**Comment by ferittuncer** [comment]
Made the requested changes and squashed the commits. @epiqueras
